src/kernels/jupyter/jupyterKernelService.node.ts

```diff
@@ -51,11 +51,11 @@ import { IJupyterKernelService } from './types';
 export class JupyterKernelService implements IJupyterKernelService {
     constructor(
         @inject(IKernelDependencyService) private readonly kernelDependencyService: IKernelDependencyService,
-        @inject(IFileSystem) private readonly fs: IFileSystem,
+        @inject(IFileSystemNode) private readonly fs: IFileSystemNode,
         @inject(IEnvironmentActivationService) private readonly activationHelper: IEnvironmentActivationService,
         @inject(IEnvironmentVariablesService) private readonly envVarsService: IEnvironmentVariablesService,
         @inject(JupyterPaths) private readonly jupyterPaths: JupyterPaths
-    ) {}
+    ) { }
```

Comment:
I keep seeing this extra space. I wonder if the linting rules for VS code enforce this and ours do the opposite?

Reply:
I was using Prettier extension and trigger it on Save. Let me see if the command line is any different.

Reply:
Fixed.